theano/tensor/sort.py

```diff
@@ -309,14 +308,19 @@ class TopKOp(theano.Op):
 
     Notes
     -----
-    - By default, this Op give two outputs: values and indices. However optimizer may
-      remove a certain output if not needed.
-
-    - Computing gradient is only possible when both values and indices are computed in
+    - The CPU and GPU op have a different order of the elements. This is
```

Comment:
I think I would rewrite this sentence with something like this: `CPU and GPU ops don't produce same output order. This is expected`.

Reply:
ok.